.gitignore

```diff
@@ -1,3 +1,8 @@
-.idea/
-.gradle/
-build/
+/.gradle/
+/.project
+/.settings
+/*/.project
+/*/.classpath
+/*/.settings
+/*/bin
+/*/build
```

Comment:
My fault (not that part with /). Fixed in https://github.com/monrealis/jinsist/commit/8715e1d9f98607c6ed6e2b71b84c0e63523f2533